### PR TITLE
Update --full option message in checkout command

### PR DESCRIPTION
**Pull request by andersonpablosilva**

### CODE REVIEW

File: docs/mlgit_commands.md [hunk 1]
```diff
@@ -133,7 +133,8 @@ Options:
   --fail-limit INTEGER RANGE  Number of failures before aborting the command.
                               This number must be in the range 0-999999999
                               [default: no limit].
-  --full                      Show all contents for each directory.
+  --full                      Show all contents for each directory when
+                              there are files to be discarded at checkout.
```
Comment (Vieira-Carlos) [inline docs/mlgit_commands.md:137]: Add
[default: False]
?

File: ml_git/commands/help_msg.py [hunk 2]
```diff
@@ -62,6 +62,7 @@
 ARTIFACT_VERSION = 'Number of artifact version to be downloaded. This number must be in the range 0-999999999 [default: latest].'
 MUTABILITY = 'Mutability type. [required]'
 STATUS_FULL_OPTION = 'Show all contents for each directory.'
+CHECKOUT_FULL_OPTION = 'Show all contents for each directory when there are files to be discarded at checkout.'
```
Comment (Vieira-Carlos) [inline ml_git/commands/help_msg.py:65]:
```suggestion
CHECKOUT_FULL_OPTION = 'Show all contents for each directory when there are files to be discarded at checkout [default: False].'
```

File: docs/mlgit_commands.md (hunk 1 above)
Comment (andersonpablosilva) [reply docs/mlgit_commands.md:137]: As it is a flag, there is no need to indicate the default value.

File: ml_git/commands/help_msg.py (hunk 2 above)
Comment (andersonpablosilva) [reply ml_git/commands/help_msg.py:65]: As it is a flag, there is no need to indicate the default value.

**Review by Vieira-Carlos — APPROVED** [review]

**Pull request merged**